spec/qiita/markdown/processor_spec.rb

```diff
@@ -1031,6 +1031,22 @@
           end
         end
       end
+
+      context "with HTML embed code for CodePen" do
+        let(:markdown) do
+          <<-EOS.strip_heredoc
+            <p data-slug-hash="foo" data-embed-version="2" class="codepen"></p>
```

Comment:
Since the `<p>` doesn't have other data-* attributes, this spec doesn't test whether data-attributes are sanitized. It should have at least data-height, data-theme-id, data-default-tab, data-user and data-pen-title attributes.